src/example-relay/src/components/TextInput.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import * as React from 'react';
+import * as sx from '@adeira/sx';
+
+import Text from './Text';
+
+type Props = {|
+  +onChange: (string) => void,
+  +value: string,
+  +placeholder?: string,
+  +label: string,
+  +name?: string,
+|};
+
+export default function TextInput({ onChange, label, ...rest }: Props): React.Node {
+  const handleChange = (e) => {
+    onChange(e.target.value);
+  };
+  return (
+    <label>
+      <Text size="small">{label}</Text>
+      <input {...rest} className={styles('input')} onChange={handleChange} />
+    </label>
+  );
+}
+
+const styles = sx.create({
+  label: {
```

Comment:
Is this style unused? 🤔 

We need to figure out some simple lint.

Reply:
Yes, it is, some linting could definitely be added at some point yes 👍

Reply:
🚀